tests/schema/test_schema.py

```diff
@@ -55,6 +55,32 @@ def test_cbiconfig_file(self):
             with self.assertRaises(ValueError):
                 toml = util._load_toml(f, "cbiconfig")
 
+    def test_analysis_file(self):
+        """schema/analysis_file"""
+
+        path = "./tests/schema/analysis.toml"
```

Comment:
Is this path relative to root? I feel like it might be worth making it relative to this test file instead.

```python
from pathlib import Path

path = Path(__file__).parent.joinpath("analysis.toml")
```

Then it doesn't matter where the test is executed. Do you think this is important at all?

Reply:
Honestly, I've not thought about this before.  I copied the convention that we're using for other tests that depend on files.  I'm open to doing it, but can we do it in a separate PR so we can update all the tests at once?

Reply:
Opened as #66.